.github/workflows/build.yml

```diff
@@ -45,10 +45,12 @@ jobs:
           cache-to: type=gha,mode=max
   build-mac:
     needs: lint
-    runs-on: macos-latest-large
+    # Apparently colima was removed from macos-13-large, so hardcode 12 instead of latest
+    runs-on: macos-12-large
```

Comment:
Doesn't the action below install it? https://github.com/douglascamata/setup-docker-macos-action/blob/main/action.yml#L57-L75

Reply:
It does, but I didn't realize it (apparently) installs it unconditionally, I assumed it would only install it if it wasn't installed already.